pyInterface/generators/generator_py.cc

```diff
@@ -256,6 +256,8 @@ void rpwa::py::exportGenerator() {
 			, &generatorWrapper::getGeneratedVertex__
 			, &generatorWrapper::default_getGeneratedVertex__
 		)
+		.def("getGeneratedXMass", &rpwa::generator::getGeneratedXMass )
+		.def("getGeneratedTPrime", &rpwa::generator::getGeneratedTPrime	)
```

Comment:
* whitespaces at end of lines
* new function definitions should not separate the two definitions for `getGeneratedVertex` (move the two new lines one line down)